package.json

```diff
@@ -42,23 +43,36 @@
     "build-fonts": "gulp fonts",
     "build-img": "gulp images",
     "build-js": "gulp javascript",
+    "build-uswds": "gulp build-uswds-if-needed",
     "check-contrast": "node ./utils/check-contrast.js",
     "clean": "gulp clean-assets",
     "crawl": "node config/crawl.js",
     "federalist": "npx gulp build",
     "lint": "gulp eslint scss-lint",
     "prestart": "gulp build",
-    "serve": "bundle exec jekyll serve --drafts --future --incremental --livereload",
+    "proof": "bundle exec htmlproofer --assume-extension --disable-external --allow-hash-href --alt-ignore '/.*/' --file-ignore '/developers/' ./_site",
```

Comment:
Add a script to do the link and HTML5 proofing. Exclude the developers docs, since they cannot be fixed directly and pull from the `uswds:main` branch. Don't let errors in that file fail tests